src/Codec/Xlsx/Parser.hs

```diff
@@ -148,14 +149,20 @@ extractSheet ar sst contentTypes wf = do
 
       condFormtattings = M.fromList . map unCfPair  $ cur $/ element (n_ "conditionalFormatting") >=> fromCursor
 
+      validations = M.fromList . map unDvPair $
+          ( ($/ element (n_ "dataValidations")) >=>
```

Comment:
that's a bit a bit unexpected way to use `xml-conduit`, why not `cur $/ element (n_ "dataValidations") &/ element (n_ "dataValidation") >=> fromCursor` ?